src/orqviz/geometric.py

```diff
@@ -148,3 +155,11 @@ def uniformly_distribute_trajectory(
     )
     eval_points = np.linspace(0, 1, num=n_points)
     return weight_interpolator(eval_points)
+
+
+def _norm_of_arrayofparametervectors(param_array: ArrayOfParameterVectors):
```

Comment:
Not sure about the name of the function...
Thoughts, @alexjuda ?

Reply:
How about just `def _norm(param_array: ArrayOfParameterVectors):`?